packages/mermaid/src/docs/.vitepress/theme/Mermaid.vue

```diff
@@ -1,4 +1,14 @@
 <template>
+  <h5>Code:</h5>
+  <div class="language-mermaid">
+    <button class="copy"></button>
+    <span class="lang">mermaid</span>
+    <pre><code contenteditable="true" @input="updateCode"  @keydown.meta.enter="renderChart" ref="editableContent" class="editable-code"></code></pre>
+    <div class="buttons-container">
+      <span>{{ ctrlSymbol }} + Enter</span><span>|</span>
```

Comment:
**question(blocking)**: What does <kbd>CTRL</kbd> + <kbd>Enter</kbd> do? I've tried it, it doesn't seem to do anything.

---

**suggestion(non-blocking)**: Use [`<kbd>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) to describe keyboard keys:

```suggestion
      <span><kbd>{{ ctrlSymbol }}</kbd> + <kbd>Enter</kbd></span><span>|</span>
```

Visually, the only difference is that `<kbd>` elements are monospaced:

### Before

![image](https://github.com/mermaid-js/mermaid/assets/19716675/9fd33e77-38ad-4c7e-87c4-a133cbfcc976)


### After

![image](https://github.com/mermaid-js/mermaid/assets/19716675/a82791c1-d6fe-408d-9591-070bae5c0edb)

Reply:
> question(blocking): What does CTRL + Enter do? I've tried it, it doesn't seem to do anything.

I thought meta would handle both command and control. Will add a separate handler for control then. 
It's supposed to render the diagram. 

---

> suggestion(non-blocking): Use [<kbd>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) to describe keyboard keys:

I had `<kbd>` initially, but it was making the meta character smaller. Instead of adding extra CSS, decided to remove kbd as the one without it looked good enough.

<img width="165" alt="image" src="https://github.com/mermaid-js/mermaid/assets/10703445/fd525cb5-9911-42a8-806e-102e070b8562">
<img width="151" alt="image" src="https://github.com/mermaid-js/mermaid/assets/10703445/c098b58b-4d95-473e-9393-2ca1dddd6f12">

---

> IMO, we could even completely remove the  ```mermaid-nocode blocks if you want.

I think @nirname added them explicitly to remove the code, let's not change that behaviour. Let me see how hard it'll be to not break it.